astropy/modeling/fitting.py

```diff
@@ -95,8 +95,9 @@ def fitter_unit_support(func):
     quantities itself. This is done by temporarily removing units from all
     parameters then adding them back once the fitting has completed.
     """
-
-    def wrapper(self, model, x, y, z=None, equivalencies=None, **kwargs):
+    @wraps(func)
+    def wrapper(self, model, x, y, z=None, **kwargs):
+        equivalencies = kwargs.pop('equivalencies', None)
```

Comment:
I wonder if it's clearer to keep `equivalences` in the signature?

Reply:
With `@wraps` the signature of `wrapper` isn't visible anymore so the documentation won't list `equivalencies` as argument. So if it were to be added it probably should be added to the individual `__call__` function signatures, that's why I added a parameter description to the docstrings there. 

But I can also undo the change here. I mean the `wrapper` already invalidates the documented signatures of the `__call__` functions by making anything except `model`, `x`, `y` and `z` a keyword-only argument.

Reply:
Just let me know what you prefer, I'll change it accordingly. :)

Reply:
Ah, OK. Thanks for the explanation. It's fine then as it is.